dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/LogProviderImpl.java

```diff
@@ -34,6 +34,12 @@ public void configure(Map<String, String> config, File logLocation, TextFileOutp
         FFDCConfigurator.init(loggingConfig);
     }
 
+    @Override
+    public void update(Map<String, Object> config) {
+        TrConfigurator.update(config);
```

Comment:
It seems all implementations of this do the same thing.  Can we not just call these two static methods ourselves?  Would that simplify things so we don't have to update the LogProvider interface?